content/filters/string_filters.md

```diff
@@ -0,0 +1,273 @@
+# String Filters
+
+String filters are used to manipulate outputs and variables of the [string](/) type.
```

Comment:
`String filters are used to manipulate string output.`
Its really weird to talk about string variables